### DISCUSSION TITLE
Music files are incorrect

### DISCUSSION THREAD

**Discussion by AliceInPain** [KRTirtho/spotube#194] [Q&A]
Hello. Im using this app for 2 days and I faced these problems:
- some song names are incorrect. Ex. Blue Sunday - the doors. It's peace frog song
- musics take too long to buffer. Even with good network
- i don't know it's because of copy right or what, but albums and playlista are not identical to spotify. I've practically memorized all my playlists, and what I imported here is different from what it was in spotify 

Android app

**Comment by ryanbuckleyca** [comment]
I noticed sometimes a title will get searched and the wrong track will be found. It would be great if there was a feature to repair with a url to correct track, as usually it's on YouTube.

**Reply by sballe73** [reply]
yes, maybe as simple as letting the youtube URL editable instead of readonly
